### PR TITLE
groovy updates: regenerate

### PR DESCRIPTION
**Pull request by bilelmoussaoui**
regenerate per the latest gir files along with a bit of cleanup & adding missing types

### CODE REVIEW

File: src/auto/dom_attr.rs [hunk 1]
```diff
@@ -28,6 +30,53 @@ glib_wrapper! {
     }
 }
 
+#[derive(Clone, Default)]
+pub struct DOMAttrBuilder {
```
Comment (antoyo) [inline src/auto/dom_attr.rs:34]: Are you sure we can create (or want to create) `DomAttr`? If no, let's remove the builders (there are probably some other types in the same situation).

Comment (bilelmoussaoui) [reply src/auto/dom_attr.rs:34]: I checked how Vala does that, and it seems to allow creating the Dom* types https://valadoc.org/webkit2gtk-web-extension-4.0/WebKit.DOM.html
Not sure if the builders makes sense though.

Comment (antoyo) [reply src/auto/dom_attr.rs:34]: Where did you see that? I could only find that [constructor](https://valadoc.org/webkit2gtk-web-extension-4.0/WebKit.DOM.Attr.Attr.html) which is `protected`.

Comment (bilelmoussaoui) [reply src/auto/dom_attr.rs]: Ah never mind. Seems like all the DOM* types have a protected constructor, I have disabled them instead.

File: src/auto/hit_test_result.rs [hunk 2]
```diff
@@ -16,6 +19,79 @@ glib_wrapper! {
     }
 }
 
+#[derive(Clone, Default)]
+pub struct HitTestResultBuilder {
```
Comment (antoyo) [inline src/auto/hit_test_result.rs:23]: I don't think it makes sense to have a builder for this type.

File: src/auto/web_hit_test_result.rs [hunk 3]
```diff
@@ -20,6 +22,88 @@ glib_wrapper! {
     }
 }
 
+#[derive(Clone, Default)]
+pub struct WebHitTestResultBuilder {
```
Comment (antoyo) [inline src/auto/web_hit_test_result.rs:26]: Same.

**Comment by antoyo** [comment]
I guess we can close this as #60 did the same?
Sorry for the delay.

**Comment by bilelmoussaoui** [comment]
yes :-)

**Pull request closed without merging**